project/src/components/comment-rating/comment-rating.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const RatingCount = {
+  PERFECT: {title: 'perfect', value: '5'},
+  GOOD: {title: 'good', value: '4'},
+  NOT_BAD: {title: 'not bad', value: '3'},
+  BADLY: {title: 'badly', value: '2'},
+  TERRIBLY: {title: 'terribly', value: '1'},
+};
```

Comment:
Немного дублирование выходит, как насчет такого варианта? И от него можно взять Object.entries. Но в целом можешь оставить
```suggestion
const ratingTitles = {
  5: 'perfect',
  ...
};
```